src/recipes/ai-context/context.ts

```diff
@@ -2,21 +2,90 @@ import { promises as fs } from 'node:fs'
 import { dirname } from 'node:path'
 
 const ATTRIBUTES_REGEX = /(\S*)="([^\s"]*)"/gim
-const BASE_URL = 'https://docs.netlify.com/ai-context'
-export const FILE_NAME = 'netlify-development.mdc'
+const BASE_URL = process.env.AI_CONTEXT_BASE_URL ?? 'https://docs.netlify.com/ai-context'
```

Comment:
How do we envision this environment being used? Should we do some validation to make sure we fail nicely if it isn't a valid URL?